Prox/Prox/Data/PlacesController.swift

```diff
@@ -47,6 +47,29 @@ class PlacesProvider {
         database.getPlace(forKey: key).upon { callback($0.successResult() )}
     }
 
+    func placeWithEvent(forKey key: String, callback: @escaping (Place?) -> ()) {
```

Comment:
for future use: perhaps rather than accessing the DB each time we need to get a place or an event, since we already have the places/events in memory to show in the UI, we could get them from in memory.

Would prevent issues where the data we get from the DB is newer than what's in memory and we try to match that new data to the old data in memory.